src/index.js

```diff
@@ -1,4 +1,6 @@
-import 'babel-polyfill';
+if (!global || !global._babelPolyfill) {
+    require('babel-polyfill')
```

Comment:
Thanks for the contribution!

I would change to `import 'babel-polyfill';` unless there's a technical reason to use require syntax. Otherwise 👍

Reply:
The if is required in case someone loads the polyfill earlier.